src/Integration.UnitTests/Suppression/SonarQubeIssuesProviderTests.cs

```diff
@@ -160,8 +159,7 @@ public void Dispose_Disconnected_TimerDisposed()
             // Arrange
             SetupSolutionBinding(isConnected: true, issues: null);
 
-            var issuesProvider = new SonarQubeIssuesProvider(mockSqService.Object, "sqkey",
-                mockTimerFactory.Object, testLogger, threadHandling.Object);
+            var issuesProvider = CreateTestSubject();
```

Comment:
This test class still uses class-level mock variables initialized in `TestInitialize`. I haven't attempted to fix that.

The only parameter that actually needs to vary in _some_ tests is the project key that is passed in.
If a test doesn't care what the project key is (like this one), then it doesn't pass one.